screens/ActivitiesScreen.tsx

```diff
@@ -47,7 +51,11 @@ const ActivitiesScreen = () => {
           SandBox to get started
         </Text>
       </TouchableOpacity>
-      <EventsList />
+      <HexColorPicker
```

Comment:
Move the event creation UI to a separate file or use storybook. We have a PR for that storybook issue btw https://github.com/tifapp/FitnessProject/pull/51